### PR TITLE
RHTAPWATCH-281: Reconfigure Splunk ClusterLogForwarders for Staging environment

### PR DESCRIPTION
**Pull request by eisraeli**
Two new indexes replaced the previous index.
Therefore there are now 2 external secrets in Vault.

### CODE REVIEW

**Review by amisstea — APPROVED** [review]

**Comment by eisraeli** [comment]
@ifireball

File: components/monitoring/logging/staging/external-secrets/kustomization.yaml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 apiVersion: kustomize.config.k8s.io/v1beta1
 kind: Kustomization
 resources:
-- monitoring-logging-splunk-log-forwarder.yaml
```
Comment (ifireball) [inline components/monitoring/logging/staging/external-secrets/kustomization.yaml:4]: Not seeing deletion for this (I suppose now no longer needed) file in the PR.

File: components/monitoring/logging/staging/external-secrets/splunk-log-forwarders-external-secrets.yaml [hunk 2]
```diff
@@ -0,0 +1,37 @@
+apiVersion: external-secrets.io/v1beta1
```
Comment (ifireball) [inline components/monitoring/logging/staging/external-secrets/splunk-log-forwarders-external-secrets.yaml:1]: Are you sure Kustomize knows how to process this kind of multi-resource file syntax? I haven't seen examples of this elsewhere.

**Review by ifireball — APPROVED** [review]
Added some questions, but overall LGTM.

File: components/monitoring/logging/staging/external-secrets/kustomization.yaml (hunk 1 above)
Comment (eisraeli) [reply components/monitoring/logging/staging/external-secrets/kustomization.yaml:4]: @ifireball I'm a bit confused. It was deleted and I also see it deleted in the PR.

File: components/monitoring/logging/staging/external-secrets/splunk-log-forwarders-external-secrets.yaml (hunk 2 above)
Comment (eisraeli) [reply components/monitoring/logging/staging/external-secrets/splunk-log-forwarders-external-secrets.yaml:1]: @ifireball I tested it with  Kustomize. It worked.

File: components/monitoring/logging/staging/external-secrets/kustomization.yaml (hunk 1 above)
Comment (ifireball) [reply components/monitoring/logging/staging/external-secrets/kustomization.yaml:4]: ok, now I see it. NVM.

File: components/monitoring/logging/staging/external-secrets/splunk-log-forwarders-external-secrets.yaml (hunk 2 above)
Comment (ifireball) [reply components/monitoring/logging/staging/external-secrets/splunk-log-forwarders-external-secrets.yaml:1]: ok. not sure where is the resolve thread button...

**Review by ifireball — APPROVED** [review]
Approved.

**Pull request merged**